backend/create_admin.py

```
@@ -0,0 +1,37 @@
+    admin_data = UserCreate(
+        username="admin",
+        email="admin@example.com",
+        full_name="System Administrator",
+        password="admin123",
+        role="system_admin",
+        is_active=True,
+        daily_capacity=8.0
+    )
+    
+    admin_user = user.create(db, obj_in=admin_data)
+    print(f"Admin user created successfully!")
```

Ruff (0.11.9)

f-string without any placeholders

Remove extraneous `f` prefix

(F541)